src/extensions/manifest.ts

```diff
@@ -60,6 +62,31 @@ export async function writeToManifest(
   await writeLocalSecrets(specs, config, options.force);
 }
 
+export async function writeEmptyManifest(
+  config: Config,
+  options: { nonInteractive: boolean; force: boolean }
+): Promise<void> {
+  if (!fs.existsSync(config.path("extensions"))) {
+    fs.mkdirSync(config.path("extensions"));
+  }
+  if (config.has("extensions") && Object.keys(config.get("extensions")).length) {
+    const currentExtensions = Object.entries(config.get("extensions"))
+      .map((i) => `${i[0]}: ${i[1]}`)
+      .join("\n\t");
+    if (
+      !(await confirm({
+        message: `firebase.json already contains extensions:\n${currentExtensions}\nWould you like to overwrite them?`,
+        nonInteractive: options.nonInteractive,
+        force: options.force,
+        default: false,
+      }))
+    ) {
+      return;
+    }
+  }
+  config.set("extensions", {});
```

Comment:
This doesn't delete the `.env` files for the extensions. Rather than trying to overwrite it, can we just throw if something already exists to keep things simple?

Reply:
This is a little awkward as is, but intentional. I'd like init to be idempotent so that it can more safely be used into scripts - init is meant as an interactive command, but some folks still try to use it non interactively.

I considered cleaning up the .env files, with a prompt, only in interactive mode, but leaving them doesn't have any ill effects.